Gemfile

```diff
@@ -55,6 +55,7 @@ group :development, :test do
   gem "debug"
   gem "minitest-excludes", "~> 2.0.1"
   gem "minitest-github_action_reporter", github: "BuonOmo/minitest-github_action_reporter", require: "minitest/github_action_reporter_plugin"
+  gem "ostruct", "~> 0.6"
```

Comment:
whats the reason for adding this?

Reply:
@grantpaulson6 ruby 3.5 stops shipping ostruct, it sends a ruby warning. And ruby warnings are errors in the rails test suite. See https://github.com/cockroachdb/activerecord-cockroachdb-adapter/actions/runs/8621739468/job/23631300792